scripts/configure-venue-access.ts

```diff
@@ -95,30 +94,16 @@ initFirebaseAdminApp(projectId)(async () => {
       console.log(
         `Setting venues/${venueId}/access/${method} to {emails: ${emails}}...`
       );
-      await admin
-        .firestore()
-        .doc(`venues/${venueId}/access/${method}`)
-        .set({
-          emails: mergeStringArrays(
-            emails,
-            (access as VenueAccessEmails).emails
-          ),
-        });
+      await accessDocRef.set({
+        emails: mergeStringArrays(emails, (access as VenueAccessEmails).emails),
+      });
```

Comment:
Instead of using `.set` and merging the emails in the script, we can use something like the following:

- https://firebase.google.com/docs/firestore/manage-data/add-data#set_a_document
  - `{ merge: true }`
- https://firebase.google.com/docs/firestore/manage-data/add-data#update_elements_in_an_array
  - `admin.firestore.FieldValue.arrayUnion`